src/library/FOSSBilling/i18n.php

```diff
@@ -27,7 +27,7 @@ public static function getActiveLocale(): string
          * Next: if that didn't work, try to use the locale in the config file
          * Finally: As a last resort, default to en_US for the locale
          */
-        return $_COOKIE['BBLANG'] ?? self::getBrowserLocale() ?: $config['i18n']['locale'] ?? 'en_US';
+        return $_COOKIE['BBLANG'] ?: self::getBrowserLocale() ?: $config['i18n']['locale'] ?: 'en_US';
```

Comment:
We've tried this and it doesn't work right. Explain why you've chosen to change this as the current method has no known bugs

Reply:
I might have been on an older version because it wasn't working for me. I've revertedit